src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -989,28 +989,26 @@ public void startService() {
     try {
       extract();
     } catch (Exception e) {
-      Logging.logError(e);
+      log.error("webgui start service threw", e);
     }
 
     start();
   }
 
   public void stop() {
     if (nettosphere != null) {
-
-      log.info("stopping nettosphere");
+      log.warn("==== nettosphere STOPPING ====");

```

Comment:
This is info level

Reply:
temporary debug - the logs are nice and clear, but still nettosphere is not behaving correctly,
when i get the issue fully resolved I'll move it to info